config/Data_Schema.toml

```diff
@@ -0,0 +1,469 @@
+[snapshot_id]
+Description = "nan"
+Deduced_Data_Type = "nan"
+Nullable = "nan"
```

Comment:
Why is nullable always "nan" 0.0 or 1.0? Is this equal to N/A, True and False?

Reply:
Yes it is. I have changed it so it always read True or False.